tests/core.py

```diff
@@ -795,26 +795,26 @@ def test_parameterized_config_gen(self):
         self.assertNotIn("{FERNET_KEY}", cfg)
 
     def test_config_use_original_when_original_and_fallback_are_present(self):
-        self.assertTrue(configuration.conf.has_option("core", "FERNET_KEY"))
-        self.assertFalse(configuration.conf.has_option("core", "FERNET_KEY_CMD"))
+        self.assertTrue(configuration.conf.has_option("core", "fernet_key"))
+        self.assertFalse(configuration.conf.has_option("core", "fernet_key_cmd"))
```

Comment:
None of these changes _should_ be needed here, and the tests should pass without, as this is how we read config values from the `airflow.*` modules, so for the sake of a smaller change we should revert all these case changes in tests.